### ISSUE TITLE
Load/Update Past EPG data

### ISSUE THREAD

**Issue by hagaygo** [kodi-pvr/pvr.iptvsimple#672]
Hi

I Noticed the addon does not seem to update past (days ?) epg data.

My IPTV's xmltv contains 3 days back and 3 days forward of data.

Since i use vod catchup which relies on epg id/start time , past epg data is important to be properly synced.

It seems the addon relies on kodi epg db to keep past data and only sync current day and forward data, so if i got a show from yesterday which was changed (time , epgid and so on) the addon will not sync it and kodi would show "old" data only.

Using latest kodi 19.4 , both on pc and librelec.

To reproduce just clear your kodi epg db and try to sync with a xmltv file that contains past 1-2 days back data , you will see that kodi will only show today's epg and forward.

P.S my kodi settings is to show 3 days back 

![image](https://user-images.githubusercontent.com/5015856/207059825-a94a032f-413d-4dbc-912f-3b3c608c860d.png)

Thanks in advance for any help.

**Comment by phunkyfish** [comment]
That is correct, as far as I know once a programme is in the past Kodi will not update it again. It’s a fair point though on catchup.

@ksooo, FYI on an interesting use case.
